orchagent/label.h

```diff
@@ -0,0 +1,80 @@
+#ifndef __LABEL__
+#define __LABEL__
+
+#include <stdint.h>
+#include <vector>
+#include <string>
+#include "converter.h"
+#include "tokenize.h"
+
+namespace swss {
+
+typedef uint32_t Label;
+
+#define LABEL_DELIMITER '/'
+#define LABEL_VALUE_MIN 0
+#define LABEL_VALUE_MAX 0xFFFFF
+
+class LabelStack
+{
+public:
+    LabelStack() = default;
+    // A list of Labels separated by '/'
+    LabelStack(const std::string &str)
+    {
+        auto labels = swss::tokenize(str, LABEL_DELIMITER);
+        for (const auto &i : labels)
```

Comment:
can you add a comment indicating where the top and bottom of label stack resides in 'vector'